build.sbt

```diff
@@ -97,19 +60,18 @@ lazy val commonSettings = Seq(
         Seq.empty
     }
   },
-  scalacOptions ++= { if (isDotty.value) Seq.empty else Seq("-target:jvm-1.8") },
   scalacOptions --= {
-    if (isDotty.value)
+    if (tlIsScala3.value)
       List(
-        "-Xfatal-warnings"
+        "-Xfatal-warnings",
+        "-source:3.0-migration"
       )
     else
       List(
       )
   },
   javaOptions ++= Seq("-Dfile.encoding=UTF8"),
-  autoAPIMappings                 := true,
-  Compile / doc / sources         := { if (isDotty.value) Seq() else (Compile / doc / sources).value }
+  Compile / doc / sources         := Seq()
```

Comment:
No docs ever again?

Reply:
What is their purpose? The official documentation is the JDK.

Reply:
I guess you're right